backends/rapidpro/msg.go

```diff
@@ -247,6 +247,11 @@ func downloadMediaToS3(ctx context.Context, b *backend, channel courier.Channel,
 		}
 	}
 
+	// to allow download media from wac when receive media message
+	if channel.ChannelType() == "WAC" {
```

Comment:
backend shouldn't know about specific channel types - can't this be set in `BuildDownloadMediaRequest` ?